rollup.config.js

```diff
@@ -29,13 +32,23 @@ const config = mode => ({
         if (source === 'ReactDOM') {
           return {id: 'react-dom', external: true};
         }
+        if (source === 'ReactNative') {
+          return {id: 'react-native', external: true};
+        }
         return null;
       },
     },
-    nodeResolve(),
+    nodeResolve({
+      extensions: target === 'native' ? ['.native.js', '.js'] : undefined,
+    }),
     commonjs(),
-    mode === 'development' ? undefined : terser({ mangle: false }),
+    mode === 'development' ? undefined : terser({mangle: false}),
   ],
 });
 
-export default [config('development'), config('production')];
+export default [
```

Comment:
It could be nice to split out the production and development builds, since I think it's rate you'd want to build both at the same time. A watch option would be great for development, too. I can imagine a `yarn dev` command. Happy to do a PR if this is something people would like to see :)